src/main/java/org/cloudfoundry/identity/uaa/api/group/UaaGroupOperations.java

```diff
@@ -81,7 +85,7 @@
 	 * @return The page of groups.
 	 * @see org.cloudfoundry.identity.uaa.api.common.model.expr.FilterRequestBuilder
 	 */
-	public PagedResult<UaaGroup> getGroups(FilterRequest request);
+	public ScimGroups getGroups(FilterRequest request);
```

Comment:
ScimGroups is a simple class extending PagedResult<ScimGroup> because I've had issues generecizing the RestTemplate calls with a generecized return object ... I've tried using ParameterizedTypeReference but had no luck. Would be open to suggestions, though.